transports/janus_mqtt.c

```diff
@@ -440,12 +532,21 @@ int janus_mqtt_init(janus_transport_callbacks *callback, const char *config_path
 	}
 
 	/* Creating a client */
-	if(MQTTAsync_create(
+	MQTTAsync_createOptions create_options = MQTTAsync_createOptions_initializer;
+
+	#ifdef MQTTVERSION_5
+		if (ctx->connect.mqtt_version == MQTTVERSION_5) {
+			create_options.MQTTVersion = MQTTVERSION_5;
+		}
```

Comment:
Indentation is broken. The way the code is structured in the rest of the repo, the `#ifdef`, `#endif`, etc. are all shifted to the left (no indentation), while what they contain should have the same indentation as if they were not wrapped in there. I won't write it under all of the changes, so please update this wherever this occurs.